examples/index.json

```diff
@@ -895,13 +895,13 @@
                 "disablePlayground": true
             },
             {
-                "name": "Basic HTTPS Listener Client",
-                "url": "basic-https-listener-client",
+                "name": "HTTPS Client",
+                "url": "https-client",
                 "disablePlayground": true
             },
             {
-                "name": "Mutual SSL",
-                "url": "mutual-ssl",
+                "name": "HTTPS Mutual SSL",
```

Comment:
Do we need this `HTTPS` word here? Isn't it better to have just  "Mutual SSL" because this itself implies that it is over HTTPS?

Reply:
Yeah. That's true.

Reply:
```suggestion
                "name": "Mutual SSL",
```